tests/callbacks/test_callbacks.py

```diff
@@ -19,6 +19,18 @@
 from tests.common import EventCounterCallback
 
 
+@pytest.fixture(autouse=True)
```

Comment:
I don't think this should be autoused. or if it is, it should case on mlflow not being installed since mlflow is an optional dep.